langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaChatModel.java

```diff
@@ -212,11 +252,20 @@ public OllamaChatModelBuilder stop(List<String> stop) {
             return this;
         }
 
+        /**
+         * @deprecated use responseFormat instead
```

Comment:
```suggestion
         * @deprecated Please use {@link #responseFormat(ResponseFormat)} instead.
         * For example: {@code responseFormat(ResponseFormat.JSON)}.
         * <br>
         * Instead of using JSON mode, consider using structured outputs with JSON schema instead,
         * see more info <a href="https://docs.langchain4j.dev/tutorials/structured-outputs#json-schema">here</a>.
```